templates/rabbitmq.config.erb

```diff
@@ -1,6 +1,9 @@
 % This file managed by Puppet
 % Template Path: <%= @module_name %>/templates/rabbitmq.config
 [
+<%- if @ssl and @ssl_versions -%>
+  {ssl, [{versions, [<%= @ssl_versions.sort.map { |v| "'#{v}'" }.join(', ') %>]}]},
+<%- end -%>
```

Comment:
Does the placement of this block make a difference with the behavior of rabbit? My original pull request had this block placed here as well, but we moved it further down in order to combine conditional statements.